### PR TITLE
make zIndex configurable for dragging layer

### PR DESCRIPTION
**Pull request by Cwallice**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6563

**Review by AndriiTsarenko — APPROVED** [review]
Hey. Thanks for your PR, and improving the component.
Seems good to me.

File: src/NestableList/API.md [hunk 1]
```diff
@@ -46,7 +46,7 @@
   | onUpdate         | func     | -            | -          | A function invoked with the new array of items whenever an item is dropped in a new location |
   | onDragStart      | func     | -            | -        | A function invoked when drag of item starts, the item's props passed as argument |
   | onDragEnd        | func     | -            | -          | A function invoked when drag of item ends, the item's props passed as argument |
-
+  | dragLayerZIndex        | number     | 100           | -          | zindex of the list item drag container |
```
Comment (shlomitc) [inline src/NestableList/API.md:49]:
```suggestion
  | dragLayerZIndex  | number     | 100           | -          | zindex of the list item drag container |
```

**Review by shlomitc — CHANGES_REQUESTED** [review]
minor change

File: src/NestableList/DragLayer.js [hunk 2]
```diff
@@ -92,7 +92,11 @@ class CustomDragLayer extends Component {
     // portal is used because of position fixed and transform issue
     return (
       <Portal>
-        <div style={layerStyles}>
+        <div
+          style={Object.assign({}, defaultLayerStyles, {
+            zIndex: dragLayerZIndex,
+          })}
```
[the remark this answers is not in the document]
Comment (Cwallice) [reply src/NestableList/DragLayer.js:98]: sure
flapped out of my mind

**Pull request merged**